### ISSUE TITLE
Generate summary table for seed-level controls

### ISSUE THREAD

**Issue by binnympaul** [RSGInc/populationsim#86]
Currently summary table are not produced for implementations with only seed level controls.
e.g., summary_PUMA.csv

**Comment by mabcal** [comment]
If we have multi-level controls (e.g region and PUMA), will it produce summaries at both levels?

**Comment by bettinardi** [comment]
Yes - it's a simple difference of controls to output at every control level, but it's so useful to have, and saves the user a couple dozen minutes of excel time each run (if I understand the question correctly)

**Comment by binnympaul** [comment]
Under the original design, PopulationSim produced summary files for all geographic levels at which controls were specified. Seed-level controls were not allowed. The summary files can be used for generating validation chart or other summaries to check control convergence at that geographic level.

For the survey weighting application, user typically specifies controls at the seed geography level. Updates were made to PopulationSim to allow controls specification at the seed geography level. However, PopualtionSim still does not produce summaries for seed level geography. Therefore, the existing scripts to produce seed-level controls validation summaries would not run.

There are two possible ways to handle this issue:
1. User can write their own post processing script to produce seed-level summary from PopulationSim outputs and inputs.
2. Add a dummy sub-seed geography which is the copy of the seed-geography. The specify the seed-level controls at the dummy sub-seed geography. This essentially translates to specifying controls at the seed-geography level but now you can configure the settings.yaml file to produce summary file at the dummy sub-seed geography.

**Comment by bstabler** [comment]
And, third, we could add summaries for seed-level controls to the software so user's don't need to do either one or two above.
